dependencies/default/pom.xml

```diff
@@ -632,12 +632,6 @@
         <scope>test</scope>
       </dependency>
 
-      <dependency>
-        <groupId>org.glassfish.jersey.core</groupId>
-        <artifactId>jersey-common</artifactId>
-        <version>${jersey.version}</version>
```

Comment:
jersey.version  properties can be delete too.

Reply:
@liubao68 I didn't mean to delete this properties. We used jersey, It's a miss delete.